offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -479,78 +481,110 @@ class OfferToPDFConverter implements OfferExporter {
-                                         <div class="col-10 cost-summary-field">Estimated net (${footerTitle}):</div>
+                                         <div class="col-10 cost-summary-field">Net (${footerTitle}):</div>
                                          <div class="col-2 price-value" id="${productGroup}-net-costs-value">12345</div>
                                      </div>
-                                     <div class="row sub-total-costs" id ="${productGroup}-sub-overhead">
-                                         <div class="col-10 cost-summary-field">Estimated overhead (${footerTitle}):</div>
-                                         <div class="col-2 price-value" id="${productGroup}-overhead-costs-value">12345</div>
-                                     </div>
-                                          <div class="row sub-total-costs" id = "${productGroup}-sub-total">
-                                          <div class="col-10 cost-summary-field">Estimated total (${footerTitle}):</div>
-                                         <div class="col-2 price-value" id="${productGroup}-total-costs-value">12345</div>
-                                         </div>
                                  </div>
                                  """
         }
 
         static String tableFooter(double overheadRatio, Affiliation affiliation){
-
             DecimalFormat decimalFormat = new DecimalFormat("#%")
             String overheadPercentage = decimalFormat.format(overheadRatio)
             double taxRatio = determineTaxCost(affiliation.country, affiliation.getCategory())
             String taxPercentage = decimalFormat.format(taxRatio)
 
             return """<div id="grid-table-footer">
+                                     <div class="col-10 cost-summary-field">Overheads (${overheadPercentage})</div>
+                                     <div class="row sub-total-costs" id="DATA_GENERATION-sub-overhead">
+                                        <div class="col-10 cost-summary-field">
+                                            Data Generation:
+                                        </div>
+                                        <div class="col-2 price-value" id="DATA_GENERATION-overhead-costs-value">
+                                            0.00
+                                        </div>
+                                     </div>
+                                     <div class="row sub-total-costs" id="DATA_ANALYSIS-sub-overhead">
+                                        <div class="col-10 cost-summary-field">
+                                            Data Analysis:
+                                        </div>
+                                        <div class="col-2 price-value" id="DATA_ANALYSIS-overhead-costs-value">
+                                            0.00
+                                        </div>
+                                     </div>
```

Comment:
is it only these two groups that have overheads?

Reply:
Correct no overhead cost is assigned to the datastorage and project management category